tools/mauve/mauve_contig_mover.pl

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/perl
```

Comment:
Can you update to use `#!/usr/bin/env perl` instead of old `#!/usr/bin/perl`. Do not want to hardcode the path where perl is located.